src/app/rundown/components/editorial-line/editorial-line.component.ts

```diff
@@ -0,0 +1,19 @@
+import {Component, HostBinding, Input} from '@angular/core';
+
+@Component({
+    selector: 'sofie-editorial-line',
+    templateUrl: './editorial-line.component.html',
+    styleUrls: ['./editorial-line.component.scss']
+})
+export class EditorialLineComponent {
+    @Input()
+    public budgetDuration: number
+
+    @Input()
+    public pixelsPerSecond: number
+
+    @HostBinding('style.left.px')
+    get left() {
```

Comment:
Explicit access modifier and return type would be nice here, such that it becomes: `public get left(): number`

Reply:
Aren't we enforcing explicit access level modifiers?

Reply:
Sounds like a missing linting rule.

Reply:
We haven't setup any linting yet for the frontend. I'll setup it up on monday.